src/chunk_dispatch.h

```diff
@@ -38,15 +39,15 @@ typedef struct ChunkDispatch
 	List *on_conflict_set;
 	List *on_conflict_where;
 	CmdType cmd_type;
-
+	ChunkInsertState *prev_cis;
+	Oid prev_cis_oid;
 } ChunkDispatch;
 
 typedef struct Point Point;
-typedef struct ChunkInsertState ChunkInsertState;
 
 extern ChunkDispatch *ts_chunk_dispatch_create(Hypertable *ht, EState *estate);
 void ts_chunk_dispatch_destroy(ChunkDispatch *dispatch);
-extern ChunkInsertState *ts_chunk_dispatch_get_chunk_insert_state(ChunkDispatch *dispatch,
-																  Point *p);
+extern ChunkInsertState *ts_chunk_dispatch_get_chunk_insert_state(ChunkDispatch *dispatch, Point *p,
+																  bool *changed_out);
```

Comment:
A) Is it okay to end variable names with prepositions? ;) 
B) It does sound a bit odd, you use cis_changed everywhere else in the PR, is there a reason we don't want to just use that name? 

But these are entirely optional. Looks good .

Reply:
changed to cis_changed_out. the out suffix is useful to know that this is a output parameter